CppSamples/DisplayInformation/RenderMultilayerSymbols/RenderMultilayerSymbols.cpp

```diff
@@ -0,0 +1,504 @@
+// [WriteFile Name=RenderMultilayerSymbols, Category=DisplayInformation]
+// [Legal]
+// Copyright 2025 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+#ifdef PCH_BUILD
+#include "pch.hpp"
+#endif // PCH_BUILD
+
+// Sample headers
+#include "RenderMultilayerSymbols.h"
+
+// ArcGIS Maps SDK headers
+#include "DashGeometricEffect.h"
+#include "Envelope.h"
+#include "GeometricEffectListModel.h"
+#include "Geometry.h"
+#include "Graphic.h"
+#include "GraphicListModel.h"
+#include "GraphicsOverlay.h"
+#include "GraphicsOverlayListModel.h"
+#include "HatchFillSymbolLayer.h"
+#include "Map.h"
+#include "MapQuickView.h"
+#include "MapTypes.h"
+#include "MultilayerPointSymbol.h"
+#include "MultilayerPolygonSymbol.h"
+#include "MultilayerPolylineSymbol.h"
+#include "PictureMarkerSymbolLayer.h"
+#include "Point.h"
+#include "PolygonBuilder.h"
+#include "PolylineBuilder.h"
+#include "SolidFillSymbolLayer.h"
+#include "SolidStrokeSymbolLayer.h"
+#include "SpatialReference.h"
+#include "StrokeSymbolLayer.h"
+#include "SymbolAnchor.h"
+#include "SymbolTypes.h"
+#include "TextSymbol.h"
+#include "VectorMarkerSymbolElement.h"
+#include "VectorMarkerSymbolLayer.h"
+#include "Viewpoint.h"
+
+// Qt headers
+#include <QColor>
+#include <QFile>
+#include <QFuture>
+#include <QImage>
+#include <QUrl>
+
+using namespace Esri::ArcGISRuntime;
+
+RenderMultilayerSymbols::RenderMultilayerSymbols(QObject *parent /* = nullptr */)
+    : QObject(parent)
+    , m_map(new Map(BasemapStyle::ArcGISLightGray, this))
+{}
+
+RenderMultilayerSymbols::~RenderMultilayerSymbols() = default;
+
+void RenderMultilayerSymbols::init()
+{
+  // Register the map view for QML
+  qmlRegisterType<MapQuickView>("Esri.Samples", 1, 0, "MapView");
+  qmlRegisterType<RenderMultilayerSymbols>("Esri.Samples", 1, 0, "RenderMultilayerSymbolsSample");
+}
+
+MapQuickView *RenderMultilayerSymbols::mapView() const
+{
+    return m_mapView;
+}
+
+// Set the view (created in QML)
+void RenderMultilayerSymbols::setMapView(MapQuickView *mapView)
+{
+  if (!mapView || mapView == m_mapView)
+      return;
+
+  m_mapView = mapView;
+  m_mapView->setMap(m_map);
+
+  // Set initial view
+  Point centerPoint(65.0,0.0,SpatialReference::wgs84());
```

Comment:
```suggestion
  Point centerPoint(65.0, 0.0, SpatialReference::wgs84());
```

spaces following commas.